airflow/providers/telegram/hooks/telegram.py

```diff
@@ -137,5 +139,5 @@ def send_message(self, api_params: dict) -> None:
         if kwargs["chat_id"] is None:
             raise AirflowException("'chat_id' must be provided for telegram message")
 
-        response = self.connection.send_message(**kwargs)
+        response = asyncio.get_event_loop().run_until_complete(self.connection.send_message(**kwargs))
```

Comment:
Would `asyncio.run()` be better here?

Reply:
I had initially used `asyncio.run()`, but updated to `get_event_loop().run_until_complete()`. Both works

Reply:
Let’s use `run` then, it has much better erro handling when something goes wrong.